### DISCUSSION TITLE
[iOS] Home screen icon - Force touch menu

### DISCUSSION THREAD

**Discussion by christianhans** [standardnotes/forum#2534] [Suggestions]
A "force touch menu" would be extremely helpful on iOS. E.g. such as Safari or the built-in Apple Notes app have it:

http://s12572.pcdn.co/wp-content/uploads/2017/02/quickActions01.jpg

This would allow quickly adding or searching for notes. I suggest 2 menu items:

* Search
* New note

**Comment by atmoio** [comment]
There's a lot of under-the-hood architectural changes that would be required before we begin thinking about this. Some of it is underway, but still some time to go.

**Comment by christianhans** [comment]
Any reason why this was closed? Still would love to see this.

**Comment by atmoio** [comment]
Repo issues are for bugs. Moved to forum for long term discussion.
